jaxb/src/main/java/com/baeldung/jaxb/dateunmarshalling/Book.java

```diff
@@ -0,0 +1,31 @@
+package com.baeldung.jaxb.dateunmarshalling;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.datatype.XMLGregorianCalendar;
+
+@XmlRootElement(name = "book")
+@XmlAccessorType(XmlAccessType.NONE)
+public class Book {
+
+    @XmlElement(name = "title", required = true)
+    private String title;
+
+    @XmlElement(name = "published", required = true)
+    private XMLGregorianCalendar published;
+
+    public Book() {
+    }
```

Comment:
Do we need this constructor? There are no other constructors, so you get the default one _for free_.